src/app/services/technical-record/technical-record.service.ts

```diff
@@ -143,9 +143,10 @@ export class TechnicalRecordService {
       select(selectRouteNestedParams),
       map(params => {
         const createdAt = params['techCreatedAt'];
-        const isProvisional = this.router.url.split('/').pop() === 'provisional';
+        const lastTwoUrlParts = this.router.url.split('/').slice(-2)
+        const isProvisional = () => lastTwoUrlParts.includes('provisional');
```

Comment:
Does it also need to be a function can it not just be in the if?

Reply:
Potentially, although I wanted to be explicit what the if statement is checking by wrapping the value in a named variable/function